lib/pbench/cli/agent/commands/results/copy.py

```diff
@@ -0,0 +1,73 @@
+import sys
+
+import click
+
+from pbench.cli.agent import pass_cli_context
+from pbench.cli.agent.options import common_options
+from pbench.cli.agent.commands.results.move import MoveResults
+
+
+@click.command()
+@common_options
+@click.option(
+    "--controller",
+    required=True,
+    envvar="_pbench_full_hostname",
+    prompt=False,
+    help="Controller name",
+)
+@click.option(
+    "--user", prompt=False, help="Pbench user (Obsolete)",
+)
```

Comment:
It's an interesting philosophical question; yes, `pbench-results-copy` and `pbench-results-move` are now tools, not easily confused with the old `pbench-move-results` and `pbench-copy-results`. Does that mean it's pointless to imply any compatibility? If someone just changes `pbench-move-results --user me` to `pbench-results-move --user me` do (or should) we care that the "obsolete" option will generate a usage message instead of being diagnosed after parsing as you're doing now?

We're not really in the position of a staged decprecation here, since we're both renaming and radically changing the behavior of the tools.

So, yeah, it can be argued that we should just drop these entirely and let them be parser errors...

Reply:
I would suggest that, if someone is in the position of changing `pbench-move-results` to `pbench-results-move`, then they are also in a position where they can remove the `--user me` option (and, of course, they are going to have to supply a token somehow, as well...).